services/kiali/1.49.1/kiali.yaml

```
@@ -42,8 +42,8 @@ data:
   name: "Kiali"
   dashboardLink: "/dkp/kiali"
   docsLink: "https://istio.io/docs/tasks/telemetry/kiali/"
-  # Chart version matches app version
-  version: "1.49.0"
+  # App version is ahead of chart version due to config updates.
+  version: "1.49.1"
```

So I think here we actually want to display the "true" AppVersion of kiali, which is 1.49.0 (1.49.1 is now our kommander application version, which is still deploying kialia appversion v1.49.0).

```suggestion
  version: "1.49.0"
```